cmd/ipsw/cmd/macho/macho_dump.go

```diff
@@ -154,6 +136,35 @@ var machoDumpCmd = &cobra.Command{
 			}
 		}
 
+		addr, err := utils.ConvertStrToInt(args[1])
```

Comment:
I'm not 💯 but the reason I wrote this is because I thought my CLI engine `cobra` couldn't do it, but I think it was due to a bug that I fixed in a PR that hasn't been merged.  So I am "mod"-ing in my PR until then.  So I think the cobra engine is actually powerful enough to take any int input and correctly convert dec/hex?

Reply:
Well the input is either an int or a string, so if cobra fails after failing to parse as an int then that won't work for this.

Reply:
I see what you mean. I don't like cmds that are magic and can take multiple type input (but you might find examples of where I've done it in the past) I prefer to make the user be explicit. So it'd be a `--section` flag

Reply:
Would you like me to change it so that a flag determines if it checks for a section name or addr. Or make `--section __TEXT.__text` cause it to not check for an addr in the arguments?

Reply:
Ya add `--section` and then some arg verification logic up top to let the user know the flags are mutually exclusive

Reply:
This is a flag that already is used by the `ipsw macho disass` cmd